tests/storage_test.go

```diff
@@ -71,15 +72,17 @@ var _ = Describe("Storage", func() {
 		var vmi *v1.VirtualMachineInstance
 		var nfsInitialized bool
 
-		initNFS := func() string {
+		initNFS := func(family k8sv1.IPFamily) string {
 			if !nfsInitialized {
 				_pvName = "test-nfs" + rand.String(48)
 				// Prepare a NFS backed PV
 				By("Starting an NFS POD")
 				os := string(cd.ContainerDiskAlpine)
-				nfsIP := tests.CreateNFSTargetPOD(os)
+				nfsIPs := tests.CreateNFSTargetPOD(os)
 				// create a new PV and PVC (PVs can't be reused)
 				By("create a new NFS PV and PVC")
+				nfsIP := libnet.GetIp(libnet.GetPodIpsStrings(nfsIPs), family)
+				Expect(nfsIP).NotTo(BeEmpty())
```

Comment:
WithOffset please (as well on the other occurrences)

Reply:
done